src/collider/GeomTree.cpp

```diff
@@ -7,7 +7,10 @@
 #include "Weld.h"
 #include "scenegraph/Serializer.h"
 
+#pragma message("FIX: warning unknown pragma")
+#ifdef __GNUC__
 #pragma GCC optimize("O3")
+#endif /* __GNUC__ */
```

Comment:
Would appreciate if this could have an MSVC branch with MSVC's optimize pragma equivalent - it's fairly important this file is optimized even in debug builds.